project-templates/ecs/basic/README.md

```diff
@@ -0,0 +1,161 @@
+# Waypoint Project Template Example
+
+This is an example of how to create and use project templating in HCP Waypoint.
+
+Pre-Requisites
+
+AWS Account with Admin Permissions
+
+Terraform Enterprise Account with No Code Module Enabled.
+
+## Initial Setup
+
+### Terraform Module Creation
+
+TODO(Teresa): Make this repo public.
+
+1. Fork the `hashicorp/terraform-aws-example-microservice-ecs-all-envs` repository in Github [here](https://github.com/hashicorp/terraform-aws-example-microservice-ecs-allenvs)
+
+
+#### Publish The Example  Module
+1. Inside Terraform Cloud, Click Registry and Select `Publish a Module`
+![img.png](../readme-images/tfc_publish_module_ss.png)
+
+
+2. Select Github under `Version Control Provider`
+
+![img_1.png](../readme-images/publish_module_vcs_ss.png)
+
+3. Select your forked repository of the no-code module created in the last step.
+
+![img_2.png](../readme-images/select_repo_ss.png)
+
+4. Click `Publish Module`
+
+![img_3.png](../readme-images/publish_module.png)
+
+5. Review the published module details
+
+![img_4.png](../readme-images/review_module_details.png)
+
+
+
+![img_8.png](../readme-images/tfc_registry_ss.png)
+
+### Terraform Cloud Workspaces
+
+1. Create terraform cloud [workspaces](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/creating) in the TFC User Interface based on the following terraform configurations provided in this example:
+        `terraform-aws-example-microservice-infra`
+        `terraform-aws-example-network`
+![img_9.png](../readme-images/create_tfc_workspace_ss.png)
+
+
+2. Go to Settings for the created TFC workspaces, Under `Remote State Sharing` allow `Share with all workspaces in this organization`.
```

Comment:
```suggestion
2. Go to Settings for the created TFC workspaces, Under `Remote State Sharing` allow `Share with all workspaces in this organization`. This will allow the no-code module to consume base infrastructure details, like the VPC id and ECS Cluster name.
```